src/pages/Thanks/ThanksPage.vue

```diff
@@ -1,20 +1,113 @@
 <template>
-	<www-page>
+	<www-page :gray-background="true">
 		<div class="row page-content">
-			<div class="small-12 columns">
-				Thanks page placeholder text
+			<div class="small-12 columns thanks" v-if="lender && loans && receipt">
+				<div class="thanks__checkout-steps">
+					<checkout-steps current-step="thanks" />
+				</div>
+
+				<div class="thanks__header">
+					<h1 class="thanks__header-h1">
+						{{ lender.firstName }}, thanks to you, {{ loans.length }}
+						{{ loans.length > 1 ? 'borrowers are' : 'borrower is' }} closer to their dreams!
+					</h1>
+					<p class="thanks__header-subhead">
+						But the journey isn't over for them and many other borrowers.<br>
+						Please tell your friends and multiply your impact
+					</p>
+				</div>
+
+				<social-share
+					class="thanks__social-share"
+					:loans="loans"
+				/>
+
+				<div class="thanks__confirmation">
+					<p>Confirmation sent to: {{ lender.email }}.</p>
+					<button @click="toggleReceipt">
+						<icon-receipt class="toggle-receipt-icon" />
+						<span>{{ isReceiptVisible ? 'Hide' : 'Show' }} Receipt</span>
+					</button>
+				</div>
+
+				<checkout-receipt
+					class="thanks__receipt"
+					v-if="isReceiptVisible"
+					:lender="lender"
+					:receipt="receipt"
+				/>
+			</div>
+
+			<div v-else>
+				No receipt data yet. TODO: pass the checkoutID to this page.
 			</div>
 		</div>
 	</www-page>
 </template>
 
 <script>
+import confetti from 'canvas-confetti';
+
+import CheckoutReceipt from '@/components/Checkout/CheckoutReceipt';
+import CheckoutSteps from '@/components/Checkout/CheckoutSteps';
+import IconReceipt from '@/assets/inline-svgs/icons/receipt.svg';
+import SocialShare from '@/components/Checkout/SocialShare';
 import WwwPage from '@/components/WwwFrame/WwwPage';
+import checkoutReceiptQuery from '@/graphql/query/checkoutReceipt.graphql';
 
 export default {
 	components: {
+		CheckoutReceipt,
+		CheckoutSteps,
+		IconReceipt,
+		SocialShare,
 		WwwPage,
-	}
+	},
+	inject: ['apollo'],
+	data() {
+		return {
+			lender: null,
+			loans: null,
+			receipt: null,
+			isReceiptVisible: true,
+		};
+	},
+	apollo: {
+		query: checkoutReceiptQuery,
+		preFetch: true,
+		preFetchVariables() {
+			return {
+				checkoutId: 38649558, // TODO, have this passed in from the checkout page
```

Comment:
Oh, probably should fix this before merge. @emuvente I see a param coming across on /thanks?kiva_transaction_id=555555... Not quite sure how to get at that though. Thoughts?

Reply:
The `preFetchVariables` function is called with the route object for the url that is being loaded, so you can access it like this:
```
preFetchVariables({ route }) {
    return {
        checkoutId: route.query.kiva_transaction_id,
    };
}
```
For the `variables` function, you can access the current route through the component instance:
```
variables() {
    return {
        checkoutId: this.$route.query.kiva_transaction_id,
    };
}
```

Reply:
Awesome thanks!